src/components/PageGating.tsx

```diff
@@ -456,21 +451,21 @@ function BuildTypeSelector() {
             onToggle={setExpanded}
             selections={buildType}
         >
-            {_.keys(BUILD_TYPE_MENU_ITEMS).map((item, index) => (
-                <SelectOption key={index} value={item} />
+            {Object.entries(BUILD_TYPE_MENU_ITEMS).map(([label, value]) => (
```

Comment:
Any substantial preferences over `_.keys` ?

Reply:
Changed to `_.map()`.